### PR TITLE
Roadmap: face detector yn face recognizer sf

### PR DESCRIPTION
**Pull request by diegohce**

### CODE REVIEW

File: objdetect.cpp [hunk 1]
```diff
@@ -175,4 +175,140 @@ bool QRCodeDetector_DetectAndDecodeMulti(QRCodeDetector qr, Mat input, CStrings*
...
+    std::vector<uchar> bufferConfigV;
+
+    uchar* bmv = (uchar*)bufferModel;
+    uchar* bcv = (uchar*)bufferConfig;
+
+
+    for(int i = 0; i < model_size; i ++) {
+        bufferModelV.push_back(bmv[i]);
+    }
+   for(int i = 0; i < config_size; i ++) {
```
Comment (deadprogram) [inline objdetect.cpp]:
```suggestion
    for(int i = 0; i < config_size; i ++) {
```

Minor formatting nit.

File: objdetect_test.go [hunk 2]
```diff
@@ -214,3 +215,268 @@ func padQr(qr *Mat) Mat {
...
+	defer rons.Close()
+
+	ronsImgSz := rons.Size()
+
+	s := image.Pt(ronsImgSz[1], ronsImgSz[0])
+
+	fd := NewFaceDetectorYN("data/face_detection_yunet_2023mar.onnx", "", s)
+	defer fd.Close()
+
+	ronsFaces := NewMat()
```
Comment (deadprogram) [inline objdetect_test.go:443]: :smile: :smile_cat: :smiley:

File: objdetect.cpp (hunk 1 above)
Comment (diegohce) [reply objdetect.cpp]: :man_facepalming: 
sorry.

**Comment by deadprogram** [comment]
@diegohce I do not think we should commit the  `data` directory with the ONNX file to the GoCV repo. Otherwise this looks good.

**Comment by deadprogram** [comment]
Thank you very much for adding this feature so quickly @diegohce !!

Now squash/merging.

**Pull request merged**